### DISCUSSION TITLE
是否可以指定浏览器内核或者版本

### DISCUSSION THREAD

**Discussion by ganjunhong** [tw93/Pake#755] [Q&A]
您好，非常棒的作品，感谢开发者们的的无私贡献
想请问下，我们在做打包的时候，
浏览器内核是怎么确定的呢。

**Comment by tw93** [comment]
浏览器内核使用的系统默认的那个

**Comment by rongrong1000** [comment]
好像 360 QQ 等小众浏览器就不支持了, 希望可以指定内核
